spark/hbase-spark/src/main/scala/org/apache/hadoop/hbase/spark/HBaseConnectionCache.scala

```diff
@@ -91,20 +91,21 @@ private[spark] object HBaseConnectionCache extends Logging {
     val tsNow: Long = System.currentTimeMillis()
     connectionMap.synchronized {
       connectionMap.foreach {
-        x => {
-          if(x._2.refCount < 0) {
-            logError(s"Bug to be fixed: negative refCount of connection ${x._2}")
-          }
+        x =>
+          {
```

Comment:
have you re-run spotless:apply after the latest commit?

still a new line here

Reply:
yes i did this still does not go :(